LICENSE

```diff
@@ -1,4 +1,255 @@
 
+This project is covered by two different licenses: MIT and Apache.
```

Comment:
Woah, this is unexpected. This changed our top level LICENSE!

Reply:
I'm not sure why/how the go get would have updated our LICENSE file. I should see why that seemed to happen. I didn't see it when I was doing things to try and fix the issues. 

As I noted in another thread, the LICENSE file change does get reverted to the common-files LICENSE file with the next update (which update_deps did).

Reply:
Also note that https://github.com/kubernetes-sigs/gateway-api/blob/main/LICENSE doesn't have this...

Reply:
> I'm not sure why/how the go get would have updated our LICENSE file.

`go get` or was it `make gen` that did it?

Reply:
Sorry for this, what I did was updating gateway-api to 1.0.0, then run `go mod tidy` and `make gen`.

Reply:
Whoops, I thought this was my PR #47699, but it's not. NVM I'm going to double-check it.

Reply:
Fixed in https://github.com/istio/istio/pull/47679

Reply:
Yes, it goes back to #47656, but still not sure what caused it. Thanks for fixing.